tests/php/features/TestFacetTypeMeta.php

```diff
@@ -262,4 +262,38 @@ public function testInvalidateMetaValuesCache() {
 	public function testInvalidateMetaValuesCacheAfterBulk() {
 		$this->markTestIncomplete();
 	}
+
+	/**
+	 * Test get_sanitize_callback method.
+	 *
+	 * @since 4.4.0
+	 * @group facets
+	 */
+	public function testGetSanitizeCallback() {
+
+		$facet_feature = Features::factory()->get_registered_feature( 'facets' );
+		$test_meta     = 'This is s test meta';
+
+		parse_str( "ep_meta_filter_new_meta_key_1={$test_meta}", $_GET );
+		$selected = $facet_feature->get_selected();
+
+		// test sanitize_text_field runs by default on taxonomy facets
+		$expected_result = sanitize_text_field( $test_meta );
+		$this->assertArrayHasKey( $expected_result, $selected['meta']['new_meta_key_1']['terms'] );
+
+		$sanitize_function = function() {
+			return 'sanitize_title';
+		};
```

Comment:
Can we just make sure we are testing something here? I was thinking of something like:

```suggestion
		$sanitize_function = function ( $function ) {
			$this->assertSame( 'sanitize_text_field', $function );
			return 'sanitize_title';
		};
```

We probably should also test if the filter is being applied in the first place (that `assertSame` wouldn't be called if the filter is not called.) Perhaps a global variable changed inside the filter and tested after the `->get_selected();` call or something similar.

Reply:
Done